presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -150,7 +152,17 @@ public SystemSessionProperties(
                         DataSize.class,
                         memoryManagerConfig.getMaxQueryMemory(),
                         true,
-                        value -> DataSize.valueOf((String) value)));
+                        value -> DataSize.valueOf((String) value)),
+                booleanSessionProperty(
+                        COLUMNAR_PROCESSING,
+                        "Use columnar processing for dictionary encoded blocks",
+                        featuresConfig.isColumnarProcessing(),
+                        false),
```

Comment:
We should have an "experimental" config flag for this, so when we roll this out, we can flip it off if we find a problem.